sdk/agrifood/agrifood-farming-rest/review/agrifood-farming.api.md

```diff
@@ -7,107 +7,169 @@
 import { Client } from '@azure-rest/core-client';
 import { ClientOptions } from '@azure-rest/core-client';
 import { HttpResponse } from '@azure-rest/core-client';
-import { PagedAsyncIterableIterator } from '@azure-rest/core-client-paging';
+import { LroEngineOptions } from '@azure/core-lro';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
 import { PathUncheckedResponse } from '@azure-rest/core-client';
-import { PollerLike } from '@azure-rest/core-client-lro';
-import { PollerOptions } from '@azure-rest/core-client-lro';
-import { PollOperationState } from '@azure-rest/core-client-lro';
+import { PollerLike } from '@azure/core-lro';
+import { PollOperationState } from '@azure/core-lro';
 import { RequestParameters } from '@azure-rest/core-client';
 import { TokenCredential } from '@azure/core-auth';
 
 // @public (undocumented)
 export interface ApplicationData {
-    applicationProductDetails?: ApplicationProductDetail[];
+    applicationProductDetails?: Array<ApplicationProductDetail>;
     area?: Measure;
     associatedBoundaryId?: string;
     attachmentsLink?: string;
     avgMaterial?: Measure;
-    createdDateTime?: Date;
+    createdDateTime?: Date | string;
```

Comment:
this feels strange to me. Presumably if it's a string, it's something we can parse into a Date, so why make the consumer have to handle both?